src/structured-data/handler.js

```diff
@@ -185,6 +185,7 @@ export async function generateSuggestionsData(auditUrl, auditData, context, scra
 
 export async function opportunityAndSuggestions(auditUrl, auditData, context) {
   const { log } = context;
+  const isInternalCustomer = context.site.getConfig().isInternalCustomer();
```

Comment:
We don't want to limit exclusion of the error message to internal customers, but customers that use AEM.

Please use `deliveryType` from the site config instead.

Example: https://github.com/adobe/spacecat-audit-worker/blob/main/src/preflight/utils.js#L41